AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
+    "- [ ] Version bounds are up to date."
```

Comment:
Not in one place yet, but I should really write up a package doc section on the subject.